connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -163,6 +164,14 @@ message Sort {
   }
 }
 
+// Relation of type [[Deduplicate]] which have duplicate rows removed, could consider either only
+// the subset of columns or all the columns.
+message Deduplicate {
+  Relation input = 1;
+  repeated string deduplicate_columns = 2;
+  bool is_all = 3;
```

Comment:
given the usage of `is_all` can't this be inferred by the list of columns? If they're identical is_all is true?

Reply:
So the question is if `is_all` is a nice convenience hack or strictly necessary, I'm ok with both :)